L1Trigger/Phase2L1ParticleFlow/python/L1NNTauProducer_cff.py

```diff
@@ -0,0 +1,22 @@
+import FWCore.ParameterSet.Config as cms
+
+L1NNTauProducer = cms.EDProducer("L1NNTauProducer",
```

Comment:
can this be replaced by `fillDescriptions` in the producer cc file?

Reply:
It could, but what would this bring?  
The same number of lines would be moved to .cc file, and then the change to the cloned "L1NNTauProducerPuppi" below would remain in the this cff file.
But I guess it is preferred so in the framework ...

Reply:
@kpedro88 If that is the preferred way I can add it in the next commit.

Reply:
Yes, using fillDescriptions is preferred.

Reply:
ok. done.